test/test_prototype_models.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import test_models as TM
+import torch
+import torchvision.prototype.models.depth.stereo.raft_stereo as raft_stereo
+from common_utils import set_rng_seed, cpu_and_gpu
+
+
+@pytest.mark.parametrize("model_builder", (raft_stereo.raft_stereo_basic, raft_stereo.raft_stereo_realtime))
+@pytest.mark.parametrize("model_mode", ("standard", "scripted"))
+@pytest.mark.parametrize("dev", cpu_and_gpu())
+def test_raft_stereo(model_builder, model_mode, dev):
+    # A simple test to make sure the model can do forward pass and jit scriptable
+    set_rng_seed(0)
+
+    # Use corr_pyramid and corr_block with smaller num_levels and radius to prevent nan output
+    # get the idea from test_models.test_raft
+    corr_pyramid = raft_stereo.CorrPyramid1d(num_levels=2)
+    corr_block = raft_stereo.CorrBlock1d(num_levels=2, radius=2)
+    model = model_builder(corr_pyramid=corr_pyramid, corr_block=corr_block).eval().to(dev)
```

Comment:
We need to use `corr_pyramid` and `corr_block` with smaller `num_levels` and `radius` to prevent `nan` in the output.

Reply:
hmmm. This means that you are not testing the original model. This is probably OK to do for now, but also it's the kind of thing we probably want to avoid on the future when we revise out testing strategy.